uui/components/datePickers/RangeDatePicker.module.scss

```diff
@@ -63,7 +63,7 @@
         }
     }
 
-    &:global(.uui-focus) {
+    &:global(.uui-focus):not(:global(.uui-invalid)) {
```

Comment:
I'm not sure if that's right, I think we should still show focus.

Reply:
restored focus